### PR TITLE
Prep community.aws 7.2.0

### PR DESCRIPTION
**Pull request by hakbailey**
##### SUMMARY
Prep release 7.2.0. Note: I updated the aws_ssm connection fix from minor change to bugfix because that seemed more accurate but lmk if it needs to be switched back.

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Review by markuman — APPROVED** [review]

**Comment by hakbailey** [comment]
recheck

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -3734,3 +3734,23 @@ releases:
...
+        for the ``aws_ssm`` connection plugin.
+    fragments:
+    - 1908-fix_find_out_bucket_region_logic.yml
+    - 1947-add_support_msk_addtinal_type.yml
+    - 2048-add-new-instance-types-in-gluejob.yaml
+    modules:
+    - description: Returns information about a Dynamo DB table
+      name: dynamodb_table_info
+      namespace: ''
+    release_date: '2024-04-02'
```
Comment (alinabuzachis) [inline changelogs/changelog.yaml:3756]:
```suggestion
    release_date: '2024-04-03'
```

**Review by alinabuzachis — APPROVED** [review]

**Comment by gravesm** [comment]
regate

**Comment by alinabuzachis** [comment]
regate

**Review by alinabuzachis — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -3734,3 +3734,24 @@ releases:
...
+    fragments:
+    - 1908-fix_find_out_bucket_region_logic.yml
+    - 1947-add_support_msk_addtinal_type.yml
+    - 20240402-lambda-test-runtime.yml
+    - 2048-add-new-instance-types-in-gluejob.yaml
+    modules:
+    - description: Returns information about a Dynamo DB table
+      name: dynamodb_table_info
+      namespace: ''
+    release_date: '2024-04-04'
```
Comment (alinabuzachis) [inline changelogs/changelog.yaml:3757]:
```suggestion
    release_date: '2024-04-05'
```

**Review by alinabuzachis — COMMENTED** [review]
I would suggest we proceed with manual merge since the main issues are basically timeouts. @tremble @markuman what do you think?

**Pull request merged**